### ISSUE TITLE
Add for authors a "List of data sets"

### ISSUE THREAD

**Issue by egonw** [fnielsen/scholia#94]
It would be nice (and support DataCite) if there was a "List of data sets". (Or, alternatively, the list of publication should have a column to indicate the type of publication.)

**Comment by fnielsen** [comment]
In the long run it might be a good idea to categorize the type of works. At the moment I think "list of data sets" will usually be empty, - and I am trying to restrict the number of queries we make.

**Comment by egonw** [comment]
Yeah, with the <iframe> approach it's hard to not show the frame at all, when there are no results...
